Localization/de-DE/home.yaml

```
@@ -84,18 +85,18 @@ Für technische Hilfe: ",
 
 #Associated text
 #"Folding refers to the way human protein folds in the cells that make up your body. We rely on the proteins to keep us healthy and they assemble themselves by folding. But when they misfold, there can be serious consequences to a person's health.",
-"Falten bezieht sich auf die Weise, wie sich menschliche Proteine in den Zellen, aus denen Ihr Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, und sie fügen sich durch das Falten zusammen. Aber wenn sie sich falsch falten kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
+"Falten bezeichnet den Vorgang, in dem sich menschliche Proteine in den Zellen, aus denen der menschliche Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, welche sich durch das Falten zusammenfügen. Wenn sich Proteine jedoch falsch falten, kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
```

The first sentence is hard to read and understand because the "...Vorgang, in dem sich..." can be misinterpreted when read quickly without deeper thinking of it.
```suggestion
"Als „Falten“ wird der Vorgang bezeichnet, wie sich die Proteine in den Zellen des menschlichen Körpers verformen. Unsere Gesundheit ist auf Proteine angewiesen und diese werden durch das Falten zusammengefügt. Wenn sich Proteine jedoch falsch falten, kann dies ernsthafte Folgen für die Gesundheit einer Person haben.",
```